.circleci/config.yml

```
@@ -0,0 +1,162 @@
+version: 2.1
+
+orbs:
+  aws-cli: circleci/aws-cli@0.1.13
+
+aliases:
+  - &workdir /home/circleci/build
+
+  - &container_config
+    docker:
+      - image: circleci/node:10
+    working_directory: *workdir
+
+  - &checkout checkout
+
+  - &install_aws_cli aws-cli/install
+
+  - &configure_aws_cli
+    aws-cli/configure:
+      aws-region: 'AWS_REGION'
+      aws-access-key-id: 'AWS_ACCESS_KEY_ID'
+      aws-secret-access-key: 'AWS_SECRET_ACCESS_KEY'
+
+  - &install_typescript_globally
```

Noticed on other repos this is misnamed, it's not installing globally. 
```suggestion
  - &install_typescript
```